### PR TITLE
413 Find Assets from Ingests

### PR DESCRIPTION
**Pull request by jasoncorum**
This adds links to a search page with all Assets associated with an ingest from the batch ingest pages.

### CODE REVIEW

File: app/views/hyrax/batch_ingest/batches/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          <%= render 'batch', presenter: Hyrax::BatchIngest::BatchPresenter.new(batch) %>
+        <% end %>
+      </tbody>
+    </table>
+    <%= paginate @batches %>
+  </div>
+<% else %>
+  <br />
+  <h4>Batches?!... Batches?!... We don't need no stinking Batches!</h4><br />
+<% end %>
```
Comment (afred) [inline app/views/hyrax/batch_ingest/batches/index.html.erb]: @jasoncorum Can we add a comment as to what's being overwritten here? This view comes from `hyrax-batch_ingest` gem, yes?

File: app/views/hyrax/batch_ingest/batches/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <th scope='col'>Error</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @presenter.batch_item_presenters.each_with_index do |batch_item_presenter, index| %>
+        <%= render 'batch_item', presenter: batch_item_presenter, index: index %>
+      <% end %>
+    </tbody>
+  </table>
+</div>
```
Comment (afred) [inline app/views/hyrax/batch_ingest/batches/show.html.erb]: @jasoncorum same... if this is an overwrite of a view coming from a gem, can we call out specifically what we're overwriting?

**Comment by jasoncorum** [comment]
@afred - i think this covers the items in your review as well as the batch data we display on  asset#show.

File: app/presenters/hyrax/asset_presenter.rb [hunk 3]
```diff
@@ -3,6 +3,8 @@
 module Hyrax
   class AssetPresenter < Hyrax::WorkShowPresenter
 
+    attr_reader :batch, :batch_url, :batch_ingest_label, :batch_ingest_date
```
Comment (afred) [inline app/presenters/hyrax/asset_presenter.rb]: @jasoncorum these aren't needed if we have explicitly created accessor methods below.

**Comment by jasoncorum** [comment]
Closing this PR without merging. Finally figured out that there was a change in Rails 5 where you now have to opt out of a belongs_to relationship being a required relationship. I have the PBCoreXML working but need to make a few more updates.

**Pull request closed without merging**